Composer/packages/client/src/shell/lgApi.ts

```diff
@@ -78,7 +77,7 @@ function createLgApi(
   return {
     addLgTemplate: updateLgTemplate,
     getLgTemplates,
-    updateLgTemplate: debounce(updateLgTemplate, 250),
+    updateLgTemplate,
```

Comment:
So I see that the debounce and throttle have been removed from the lgApi and luApi respectively. Don't we still want to debounce and throttle them despite the lg and lu parsers being fast now for performance reasons? This is gonna trigger an updateLgTemplate or luTemplate on every keystroke change in the lg file right? Either we can do it here or debounce, throttle inside the dispatcher factory functions returns. What do u think?